doc/imv.1

```diff
@@ -1,131 +1,125 @@
-.TH IMV 1 "" "" "multimedia"
-.SH NAME
-imv \- image viewer
-.SH SYNOPSIS
-.nf
-\fBimv\fP [-rfaudlh] [-n <NUM|PATH>] [-b COLOR] [-e FONT:SIZE] [-t SECONDS] [images...]
-.fi
-.sp
-.SH DESCRIPTION
-.sp
-\fBimv\fP is a X11/Wayland image viewer aimed at users of tiling window managers.
+.Dd $Mdocdate: December 30 2015$
+.Dt IMV 1
+.Os
+.Sh NAME
+.Nm imv
+.Nd view images
+.Sh SYNOPSIS
+.Nm
+.Op Fl adfhlru
+.Op Fl b Ar color
+.Op Fl e Ar font:size
+.Op Fl n Ar position
+.Op Fl t Ar seconds
+.Op Ar
+.Sh DESCRIPTION
+.Nm
+is an image viewer for
+.Xr X 7
+and Wayland, aimed at users of tiling window managers.
 It supports a wide variety of image file formats, including animated gif files.
-.SH FLAGS
-.TP
-.B -
-Read paths from stdin. One path per line.
-.TP
-.B -r
-Recursively search input paths for files.
-.TP
-.B -f
-Start in full screen mode
-.TP
-.B -a
+.Nm
+reloads current image if it detects changes to the file.
+.Pp
+When run with argument
+.Ar - ,
+.Nm
+reads the list of images from standard input.
+.Pp
+The options are as follows:
+.Bl -tag -width Ds
+.It Fl a
 Default to showing images at actual size.
-.TP
-.B -u
-Use nearest neighbour resampling. Recommended for pixel art.
-.TP
-.B -d
-Show overlay.
-.TP
-.B -l
+.It Fl b Ar color
+Set the background color.
+Either
+.Ar checks
+or a hex color value.
+Defaults to
+.Ar #000000 .
+.It Fl e Ar font:size
+Set the font used by the overlay.
+.Ar font
+can be any valid system font, such as FreeSans, FreeMono, etc.
+Defaults to
+.Ar Monospace:24 ,
+which is default monospace font, as configured in
+.Xr fonts.conf 5 .
+.It Fl d
+Show textual overlay.
+.It Fl f
+Start in full screen mode.
+.It Fl h
+Print help message.
+.It Fl l
 List open files on exit.
-.TP
-.B -h
-Print help.
-.SH OPTIONS
-.TP
-.BI "-n " <NUM|PATH>
-Start at image number NUM, or the image with the path PATH.
-.TP
-.BI "-b " BG
-Set the background color. Either 'checks' or a hex color value. Defaults to '#000000'.
-.TP
-.BI "-e " FONT:SIZE
-Set the font used by the overlay. FONT can be any valid system font, such as
-FreeSans, FreeMono, etc.  Defaults to "Monospace:24", which is default monospace font, as configured in
-.BR fonts.conf (5).
-.TP
-.BI "-t " SECONDS
-Set the slideshow delay in seconds. Setting this to zero disables slideshow
-mode, which is the default.
-.SH CONTROLS
-.sp
-imv can be controlled using the mouse or keyboard.
-.TP
-.B Drag + Left Mouse Button
-Pan.
-.TP
-.B MouseWheel
+.It Fl n Ar position
+Start at specific position in image list.
+.Ar position
+may be expressed as a number or as a path of an image.
+.It Fl r
+Recursively search input paths for files.
+.It Fl t Ar seconds 
+Set the slideshow delay in seconds.
+Setting this to zero disables slideshow mode, which is the default.
+.It Fl u
+Use nearest neighbour resampling. Recommended for pixel art.
+.El
+.Sh CONTROLS
+.Bl -tag -width Ds
+.It Aq Cm left mouse button
+Pan in drag direction.
```

Comment:
Very nitpicky here, but if I click and drag left, I pan right. I'm not sure if "in drag direction" is an apt description.

Perhaps "Pan relative to mouse motion" or similar would be more correct?